homeassistant/components/awair/config_flow.py

```diff
@@ -119,12 +119,16 @@ async def async_step_cloud(self, user_input: Mapping[str, Any]) -> ConfigFlowRes
     def _get_discovered_entries(self) -> dict[str, str]:
         """Get discovered entries."""
         entries: dict[str, str] = {}
-        for flow in self._async_in_progress():
-            if flow["context"]["source"] == SOURCE_ZEROCONF:
-                info = flow["context"]["title_placeholders"]
-                entries[flow["context"]["host"]] = (
-                    f"{info['model']} ({info['device_id']})"
-                )
+
+        flows = cast(
+            set[Self],
+            self.hass.config_entries.flow._handler_progress_index.get(DOMAIN) or set(),  # noqa: SLF001
```

Comment:
This is the third place that this protected property is called: `self.hass.config_entries.flow._handler_progress_index.get`
Should there be a public method available?

Reply:
Not at this time, we don't want to encourage accessing flows directly, it should be an exceptional case.